modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/home/page-definition.json

```diff
@@ -21,7 +21,7 @@
 						"definition": {
 							"fragment": {
 								"key": "public-sites-navigation",
-								"siteKey": "CP"
+								"siteKey": "[$GROUP_KEY$]"
```

Comment:
what does swapping `CP` to `[$GROUP_KEY$]` do?

Reply:
"CP" is a static value, shouldn't this value be variable according to the site created?

FYI: This change has no connection with the implementation of the banner, it would be a code improvement.